src/sys/unix/selector/io_source/stateless.rs

```diff
@@ -0,0 +1,47 @@
+use crate::{Interest, Registry, Token};
+use std::io;
+use std::os::unix::io::RawFd;
+
+pub(crate) struct IoSourceState;
+
+impl IoSourceState {
+    pub fn new() -> IoSourceState {
```

Comment:
Nit: also for the others.
```suggestion
    pub(crate) fn new() -> IoSourceState {
```